src/coll/mod.rs

```diff
@@ -601,7 +602,29 @@ impl Collection {
         pipeline: impl IntoIterator<Item = Document>,
         options: Option<ChangeStreamOptions>,
     ) -> Result<ChangeStream> {
-        todo!();
+        let pipeline: Vec<Document> = pipeline.into_iter().collect();
+        let mut options = options;
```

Comment:
Ditto above about marking the parameter as `mut`

Reply:
fixed

Reply:
This doesn't seem to be fixed in the latest commit.

Reply:
didn't push earlier. fixed now